core-services/prow/02_config/openshift/cert-manager-operator-release/_prowconfig.yaml

```diff
@@ -10,16 +10,6 @@ branch-protection:
               protect: true
             release-1.15:
               protect: true
-              required_status_checks:
-                contexts:
```

Comment:
Does using just `- Red Hat Konflux` help here?
I understand the loopy behaviour but wanted to keep some checks so it's easier in the future to automate merge(s), say for the reference update PRs by the konflux bot; without required checks it can be harder and can break unknown things in between multiple PR merges.